lib/ansible/modules/network/meraki/meraki_static_route.py

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Kevin Breit (@kbreit) <kevin.breit@kevinbreit.net>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: meraki_static_route
+short_description: Manage static routes in the Meraki cloud
+version_added: "2.8"
+description:
+- Allows for creation, management, and visibility into static routes within Meraki.
+
+options:
+    auth_key:
+        description:
+        - Authentication key provided by the dashboard. Required if environmental variable MERAKI_KEY is not set.
```

Comment:
Make this two lines. The first line is always a summary of the parameter.
Next options clarify its use, format, etc...
```suggestion
        - Authentication key provided by the dashboard.
        - Required if environment variable C(MERAKI_KEY) is not set.
      type: str
```